javascript/example-customer-account--wishlist--ts/README.md

```diff
@@ -0,0 +1,21 @@
+# Customer account UI Extension
+
+## Prerequisites
+
+Before you start building your extension, make sure that you’ve created a [development store](https://shopify.dev/docs/apps/tools/development-stores) with the [Checkout and Customer Accounts Extensibility](https://shopify.dev/docs/api/release-notes/developer-previews#previewing-new-features).
+
+## Your new Extension
+
+Your new extension contains the following files:
+
+- `README.md`, the file you are reading right now.
+- `shopify.extension.toml`, the configuration file for your extension. This file defines your extension’s name.
+- `src/*.tsx`, the source code for your extension.
```

Comment:
so here, for all readme, do we want to change it to `src/*.ts` since this is just vanilla ts?